ts/src/coinex.ts

```diff
@@ -2647,7 +2647,7 @@ export default class coinex extends Exchange {
         let address = undefined;
         let tag = undefined;
         const partsLength = parts.length;
-        if (partsLength > 1) {
+        if (partsLength > 1 && parts[0] !== 'cfx') {
```

Comment:
@pcriadoperez Did you test this? Also can you elaborate a bit more why this is needed?